lib/resources/dstu2/allergy_intolerance.yaml

```diff
@@ -0,0 +1,219 @@
+---
+field_name_base_url: http://hl7.org/fhir/DSTU2/allergyintolerance-definitions.html#AllergyIntolerance
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  example: AllergyIntolerance
+  note: resourceType must be AllergyIntolerance
+  url: http://hl7.org/fhir/DSTU2/json.html#resources
+
+- name: id
+  required: 'Yes'
+  cardinality: 0..1
+  type: id
+  description: The logical id of the resource to update.
+  example: '{"id": "123412"}'
+  note: The id value must match the AllergyIntolerance/&lt;id> value.
+  action: update
+
+- name: contained
```

Comment:
Is this intended to be a link to the detailed allergy fields? Contained isn't one of them, so it just goes to the top of that resource. There is a section in the spec about contained resources: http://hl7.org/fhir/DSTU2/references.html#contained